dashboard/cron.py

```diff
@@ -11,6 +11,7 @@
 from django.db.models import QuerySet
 from django_cron import CronJobBase, Schedule
 from google.cloud import bigquery
+from pandas import DataFrame
```

Comment:
To be consistent with the rest of the code, please remove this import and use `pd.DataFrame`